components/ai_chat/core/browser/ai_chat_database.cc

```diff
@@ -196,6 +196,7 @@ std::vector<mojom::ConversationPtr> AIChatDatabase::GetAllConversations() {
     conversation->has_content = true;
 
     conversation->associated_content = mojom::SiteInfo::New();
+    conversation->associated_content->content_id = -1;
```

Comment:
`-1` should be a constant defined in the mojom file. Invalid tab id or something.

Reply:
Wow, I had no idea you could actually just define constants in mojom :open_mouth: